brokerapi/brokers/spanner/broker.go

```diff
@@ -246,12 +282,32 @@ func (s *SpannerBroker) Deprovision(instanceID string, details models.Deprovisio
 	})
 
 	if err != nil {
-		return fmt.Errorf("Error creating client: %s", err)
+		deleteErr := createFakeDeleteOperation(instanceID, instance.ServiceId, err.Error())
```

Comment:
Because Spanner doesn't really use operations for delete - it runs deletes synchronously. But CF doesn't know services can be half async, so I have to pretend it's entirely async. And it simplifies the flow to use a fake operation entry to do that. Will add a comment to clarify.